manage4.js

```diff
@@ -225,10 +321,10 @@ export async function main(ns) {
             }
 
             var global_id = global_filler_id++
-            if (0 === ns.exec(sc_weak, server, th_weak, target, global_id)) {
-                ns.tprint(sprintf("Failed to exec weak(%s,%s,%s,%s)", sc_weak, server, th_weak, target))
-                return
-            }
+                if (0 === ns.exec(sc_weak, server, th_weak, target, global_id)) {
+                    ns.tprint(sprintf("Failed to exec weak(%s,%s,%s,%s)", sc_weak, server, th_weak, target))
+                    return
+                }

```

Comment:
This formatting is off -- there are probably accidental tabs here. Please reformat to spaces.